src/main/java/com/sergiofah/integracao/service/Service.java

```diff
@@ -0,0 +1,29 @@
+package com.sergiofah.integracao.service;
+
+import org.springframework.web.reactive.function.client.WebClient;
+
+public class Service {
+    private WebClient webClient;
+    public Service() {
+        this.webClient = WebClient
+                .builder()
+                .baseUrl("http://localhost:8080")
+                .build();
+    }
+    public WebClient getWebClient() {
+        return webClient;
+    }
```

Comment:
It's necessary to separate class declaration, attribute declaration, constructor declaration and first method declaration.